README.md

```diff
@@ -4,12 +4,14 @@ Building right now may be a bit rough, we are improving and finding ways to simp
 
 ## Linux (generic)
 
-1. (if you want version with tensorflow) Install `tensorflow_cc` by following steps described [here](https://github.com/FloopCZ/tensorflow_cc).
-2. (if you want cuDNN version) Install CUDA and cuDNN.
-3. Install ninja build (`ninja-build`), meson, and (optionally) gtest (`libgtest-dev`).
-4. Go to lc0/
-5. Run ./build.sh
-6. `lc0` will be in `lc0/build/release/` directory
+1. Install backend:
+    - (if you want version with tensorflow) Install `tensorflow_cc` by following steps described [here](https://github.com/FloopCZ/tensorflow_cc).
+    - (if you want cuDNN version) Install CUDA and cuDNN.
```

Comment:
Maybe add cuda/cudnn links to nvidia website?

Reply:
Good idea, done.